src/syntax/read.rkt

```diff
@@ -40,7 +40,14 @@
         ['() '()]
         [(list prop val rest ...) (cons (cons prop val) (loop rest))])))
 
-  (define default-prec (dict-ref prop-dict ':precision
+  (define prop-dict* ; override fpcore props
+    (let loop ([prop-dict prop-dict] [ctx override-ctx])
+      (match ctx
+       ['() prop-dict]
+       [(list (cons prop val) rest ...)
+        (loop (dict-set* prop-dict prop val) rest)])))
+  
+  (define default-prec (dict-ref prop-dict* ':precision
                                  (if (flag-set? 'precision 'double)
```

Comment:
I think now that this flag is deprecated for a release or two, we can drop it entirely.